src/platform/datapath_raw_xdp_win.c

```diff
@@ -1882,3 +1882,115 @@ CxPlatIoXdpShutdownEventComplete(
         Partition);
     CxPlatDpRawRelease((XDP_DATAPATH*)Partition->Xdp);
 }
+
+_IRQL_requires_max_(PASSIVE_LEVEL)
+QUIC_STATUS
+CxPlatDataPathGetRssConfig(
+    _In_ uint32_t InterfaceIndex,
+    _Outptr_ _At_(*RssConfig, __drv_allocatesMem(Mem))
+        CXPLAT_RSS_CONFIG** RssConfig
+    )
+{
+    QUIC_STATUS Status = QUIC_STATUS_SUCCESS;
+    uint32_t RssConfigSize = 0;
+    XDP_RSS_CONFIGURATION *RawRssConfig = NULL;
+
+    if (RssConfig == NULL) {
+        Status = QUIC_STATUS_INVALID_PARAMETER;
+        goto Error;
+    }
```

Comment:
If this is an internal function, we can probably assert this instead of checking it. SAL indicates this is a required output too.

Reply:
Well it's part of the CxPlat interface, so if we make that library separate, it should validate this

Reply:
If we ever actually do that, then there will be plenty to clean up. For now, simpler is better IMO.

Reply:
I think I'm going to leave this here. Why would we want less validation? It's on the control path, so this isn't performance sensitive.

Reply:
I would prefer a DBG assert instead of a check for all internal functions, independent of if it _might_ be exposed later.